### ISSUE TITLE
[a11y][Android] TalkBack focus border is incorrectly positioned when device has a notch on left side of the screen

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#67835]
b/170498977 

https://github.com/flutter/engine/blob/48687ca3ca6e4f24281ea998789cc21ad840accb/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L1311

Flutter's AccessibilityBridge unconditionally uses the left system insets to shift highlights in case the version of Android (with OEM customizations) places the navigation bar on the left edge of the screen. This should be disabled on Android Q+ when layoutInDisplayCutoutMode is non-default.

![image](https://user-images.githubusercontent.com/6687929/95661580-47017b00-0ae5-11eb-9a26-05ed10fa7ba2.png)

![image](https://user-images.githubusercontent.com/6687929/95661582-4cf75c00-0ae5-11eb-920b-e36b1592d6bb.png)

**Comment by HansMuller** [comment]
CC @chunhtai
